tests/test_connector.py

```diff
@@ -2031,8 +2031,21 @@ async def handler(request):
     await session.close()
 
 
+@pytest.mark.parametrize(
+    "host",
+    [
+        "127.0.0.1",
+        "localhost",
+        "localhost.",
+    ],
+    ids=(
+        "ip address",
+        "domain name",
+        "fully-qualified domain name",
+    ),
```

Comment:
It's usually better to match ids with params when they are coupled together. When the list grows, it's harder to visually match them...
```suggestion
    (
        pytest.param("127.0.0.1", id="ip address"),
        pytest.param("localhost", id="domain name"),
        pytest.param("localhost.", id="fully-qualified domain name"),
    ),
```

Reply:
That looks better, thanks!